Update contribute.md to be more doc specific (#2024)
centreon/centreon-documentation: versioned_docs/version-22.10/resources/contribute.md

You can contribute to the Centreon documentation by submitting pull requests on [our GitHub repository](https://github.com/centreon/centreon-documentation). As an external contributor, you cannot commit directly to the repository: a fork will be created automatically when you commit changes.

This page only details the procedure to request changes in the documentation:
* If your problem is a general question about Centreon or the documentation, ask in the [Slack](https://centreon.github.io/register-slack/) channel instead, or on our community platform [The Watch](https://thewatch.centreon.com/).

> * If you have an idea for a new feature please submit it or on the [Ideas](https://thewatch.centreon.com/ideas) section of The Watch.

Reviewer: ```suggestion
* If you have an idea for a new feature please submit it in the [Ideas](https://thewatch.centreon.com/ideas) section of The Watch.
```
